src/app/item-card/item-card.component.scss

```diff
@@ -17,7 +17,3 @@ mat-card {
   margin-top: 10px;
   margin-bottom: 10px;
 }
-
-.center-text {
```

Comment:
This css class is meant to center the texts when the application goes into mobile view.

I think we should add this class back into the file.

If not, then there's no point in keeping `[ngClass]="{'center-text': windowService.width < mobileWidth}"` in `<mat-card>`

Add this class back in and check out the application again by making the browser smaller, then larger to see the effect.